### ISSUE TITLE
[Bug]: Installing the weights for the model

### ISSUE THREAD

**Issue by ThiagoRBM** [Sygil-Dev/sygil-webui#1665]
### What happened?

Hello,
I got the message "Now you need to install the weights for the stable diffusion model." after instaling the webui, but the link (https://sd-webui.github.io/stable-diffusion-webui/docs/2.linux-installation.html#initial-start-guide) gives back a 404 error.
Am I doing something wrong?
Apparently the prior steps run without issues.

Thanks!

### Version

0.0.1 (Default)

### What browsers are you seeing the problem on?

Firefox

### Where are you running the webui?

Linux

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by cstueckrath** [comment]
yeah, not a bug but a misplaced link. Go here: https://github.com/Sygil-Dev/sygil-webui/blob/master/docs/1.Installation/3.linux-installation.md
